editor/src/clj/editor/debug_view.clj

```diff
@@ -611,31 +612,59 @@
             (current-session debug-view evaluation-context))
   (run [debug-view] (mobdebug/exit! (current-session debug-view))))
 
-(defn- can-change-resolution? [debug-view prefs evaluation-context]
-  (and (targets/controllable-target? (targets/selected-target prefs))
-       (not (suspended? debug-view evaluation-context))))
+(defn- should-rotate-device? [prefs workspace]
+  (prefs/get-prefs prefs (prefs/make-project-specific-key "should-rotate-device" workspace) false))
 
-(def should-rotate-device?
-  (atom false))
+(defn- change-resolution! [prefs width height workspace]
+  (let [target (targets/selected-target prefs)
+        should-rotate-device? (should-rotate-device? prefs workspace)]
+    (when target
+      (if (targets/all-launched-targets? target)
+        (doseq [launched-target (targets/all-launched-targets)]
+          (engine/change-resolution! launched-target width height should-rotate-device?))
+        (engine/change-resolution! target width height should-rotate-device?)))))
```

Comment:
In case we have multiple targets launched, apply simulated resolution to all of them